src/components/NavBar.js

```diff
@@ -46,11 +64,18 @@ export default function Navbar() {
         </div>
         <nav className={navbar_menu_container}>
           <ul className={nav_menu}>
-            {navItems.map((buttonText, index) => (
+            {navItems.map((navItem, index) => (
               <li className={nav_item} key={index}>
-                <Link to={buttonText} smooth={true} offset={-navBarHeight} duration={1000}>
-                  <div className={nav_links}>{buttonText}</div>
-                </Link>
+                <div className={nav_item_div}>
+                  <NavLink
+                    to={navItem.urlPath}
+                    className={(isActive) => {
```

Comment:
`isActive.isActive` is ugly to read. try to destructurise this param like this:

```suggestion
                    className={( { isActive } ) => {
```